test/lima.yaml

```diff
@@ -0,0 +1,30 @@
+images:
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release/ubuntu-24.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/24.04/release/ubuntu-24.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
+cpus: 2
+memory: "2g"
+plain: true
+networks:
+- socket: /var/run/socket_vmnet
+provision:
+- mode: system
+  script: |
+    #!/bin/bash
+    set -eux -o pipefail
+    command -v ipref3 >/dev/null 2>&1 && exit 0
+    export DEBIAN_FRONTEND=noninteractive
+    apt-get update
+    apt-get install -y iperf3
```

Comment:
We can start iperf3 service here so no additional setup is needed during testing.

We need it only one one vm, but it does not make much difference when not used.